.eslintignore

```diff
@@ -2,6 +2,7 @@ core/templates/expressions/parser.js
 core/templates/google-analytics.initializer.ts
 backend_prod_files/*
 core/tests/protractor.conf.js
+core/tests/wdio.conf.js
```

Comment:
Any specific reason for ignoring this file from the lint test?

Reply:
Done. Resolved now